nuxeo-user-group-management/nuxeo-user-management.html

```diff
@@ -167,6 +167,7 @@
       nuxeo-dialog {
         min-width: 400px;
         padding-top: 24px;
+        text-transform: initial;
```

Comment:
Don't forget that applying this text transformation will affect all `nuxeo-dialog`s in the element, which I'm not sure it's what we want since the ticket only complains about the delete confirmation. Would be good to review this change with either Tiago and/or Isabel.